install.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+
+require('shelljs/global');
+var gup = require('guppy-cli');
+var async = require('async');
+var pack = require(rootApplicationPath() + '/package.json');
+
+// Install each confgured hook.
+pack['guppy-hooks'] && async.each(pack['guppy-hooks'], function (hook, next) {
```

Comment:
Consider it done. Do you have any applicable code-style which I should follow for further contributions?